frontend/src/components/SaveButton.tsx

```diff
@@ -49,7 +49,8 @@ const SaveButton = ({
       >
         {loading ? (
           <>
-            <Spinner className="height-3 width-3" /> {loadingText}
+            <Spinner className="height-105 width-105 button-icon-large" />{" "}
```

Comment:
This fixes the loading spinner changing the button size. The spinner is now the same size as the star icon:

<img width="454" alt="image" src="https://github.com/user-attachments/assets/5c59a37b-2b3c-4d08-b025-efcdbfa90d93" />

vs:

<img width="475" alt="image" src="https://github.com/user-attachments/assets/f4f1c1e2-9a54-4b81-b1e5-658ef4674b1c" />